packages/fxa-content-server/app/scripts/models/user.js

```diff
@@ -713,17 +713,17 @@ var User = Backbone.Model.extend({
    * Should the model be initialized using browser data?
    *
    * @param {Object} service service being signed into.
-   * @param {Boolean} isDevicePairingAsAuthority
+   * @param {Boolean} isPairing device is trying to pair
    * @returns {Boolean}
    */
-  shouldSetSignedInAccountFromBrowser (service, isDevicePairingAsAuthority) {
-    // If service=sync or the device is pairing as the authority,
+  shouldSetSignedInAccountFromBrowser (service, isPairing) {
```

Comment:
This makes sense to me. To have access to the start pairing button, the browser should have gone through at least an email verification. If the session has been email verified then it would be able to call the `checkTotpStatus` endpoint and see if the account has TOTP.